data/faq.html

```diff
@@ -354,38 +405,45 @@ <h3 lang="es">Navegación, rutas e itinerarios</h3>
 <dd lang="es">Confirma que se haya establecido tu posición; a continuación, elige con el dedo el punto del mapa al que quieras ir. Toca el icono «Ruta» en la ventana emergente. Para cambiar el tipo de ruta, pulsa los iconos «A pie» o «En coche» situados en la parte superior de la pantalla. Si quieres elegir otra ubicación de inicio, pulsa el icono «Ruta» en la pantalla principal. Aparecerá la ventana de Planificación de ruta. Pulsa en «Desde» para seleccionar un punto de partida y pulsa en «A» para un punto de destino. Presiona el botón «Inicio» para empezar a moverte (la navegación solo está disponible cuando se crea una ruta desde tu ubicación actual). Pulsa «Empezar» para empezar a moverte. Pulsa la «X» situada en la esquina superior izquierda de la pantalla para finalizar la ruta.</dd>
 
 <dt lang="en">How can I manage voice instructions settings?</dt>
 <dt lang="ru">Настройки голосовой навигации.</dt>
 <dt lang="de">Wie kann ich die Einstellungen für die Sprachausgabe ändern?</dt>
 <dt lang="es">¿Cómo puedo administrar los Ajustes de las instrucciones de voz?</dt>
-<dd lang="en">Open app’s settings > Navigation. Here you can choose the language and turn off voice instructions. Please note that we don’t use pre-recorded voices but TTS (text-to-speech) engine. It means that the output quality of voice and language selection depends on the TTS engine installed on your device.</dd>
-<dd lang="ru">Откройте настройки приложения > Навигация. Здесь вы можете поменять язык, а также выключить голосовые подсказки. Обратите внимание: мы не используем заранее записанный голос – подсказки озвучиваются с помощью функции воспроизведения текста. Это значит, что качество голоса и список доступных языков зависят от синтезатора речи, установленного на вашем устройстве.</dd>
-<dd lang="de">Öffnen Sie App-Einstellungen > Navigation. Hier können Sie die Sprache ändern und die Sprachausgabe deaktivieren. Beachten Sie, dass wir keine vorher aufgenommenen Stimmen nutzen, sondern TTS-Software (Text-to-Speech). Daher können die Ausgabequalität der Stimme und die Sprachauswahl von den TTS-Einstellungen Ihres Gerätes abhängen.</dd>
-<dd lang="es">Abre los Ajustes de la aplicación > Navegación. Aquí puedes elegir el idioma y desactivar las instrucciones de voz. Ten en cuenta que no usamos voces pregrabadas sino un motor TTS (conversión de texto a voz). Por ello, la calidad de producción de la selección de idioma y voz depende del motor TTS instalado en tu dispositivo.</dd>
+<dd lang="en">Open app’s settings –> Navigation. Here you can choose the language and turn off voice instructions. Please note that we don’t use pre-recorded voices but TTS (text-to-speech) engine. It means that the output quality of voice and language selection depends on the TTS engine installed on your device.</dd>
```

Comment:
Open the app's settings

... but TTS (text-to-speech) engines. 

the voice output quality and language availability